### PR TITLE
Add MemoryReservation support

### PR DESCRIPTION
**Pull request by the40san**
closes aws/amazon-ecs-cli#145

Added MemoryReservation option for ecs-cli.

Any docker-compose file format has no `memory_reservation` directive, but Amazon ECS has it.
I think this is effective for production environment.

Could you review this please?

### CODE REVIEW

File: ecs-cli/vendor/github.com/docker/libcompose/config/schema.go [hunk 1]
```diff
@@ -371,7 +372,8 @@ var servicesSchemaDataV2 = `{
         "volumes": {"type": "array", "items": {"type": "string"}, "uniqueItems": true},
         "volume_driver": {"type": "string"},
         "volumes_from": {"type": "array", "items": {"type": "string"}, "uniqueItems": true},
-        "working_dir": {"type": "string"}
+        "working_dir": {"type": "string"},
+        "memory_reservation": {"type": "integer"}
```
Comment (the40san) [inline ecs-cli/vendor/github.com/docker/libcompose/config/schema.go]: I will fix this. Thank you.

**Comment by c100k** [comment]
Support for version 2 : https://github.com/docker/compose/pull/4563

**Comment by the40san** [comment]
Thank you. I'll start work to fix this.

**Comment by yinshiua** [comment]
@the40san Looks like someone has opened up a PR to use the updated libcompose mem_reservation. :) PR https://github.com/aws/amazon-ecs-cli/pull/228

**Pull request closed without merging**